web/react-web/src/Content/Graph/Graph.js

```diff
@@ -142,9 +159,32 @@ class Graph extends Component {
                 name = `Active cases for ${one.long_name}`;
             }
 
+            let yRatio = 1.0;
+            switch (dataScaling) {
+                case SCALING_SAME_PEAK:
+                    let maxValue = one.max_value;
+                    if (one.type !== 'prediction') {
+                        maxValue = Math.max(...one.values);
+                    }
+                    yRatio = maxPeakValue / maxValue / maxPeakValue * 100.0;
```

Comment:
Something is off here, the `maxPeakValu` cancels out and `yRatio` is just `maxValue` multiplied by 100?
```suggestion
                    yRatio = maxValue * 100.0;
```